File: Python/5_CalculoIMC.py
def CalculaIMC(peso, altura):
    total = peso / (altura * altura)

    if total < 18.5:
        return "Seu IMC é de : ", total, " Magro"
    elif 18.5 > total <= 25.00:
        return "Seu IMC é de : ", total, " Normal"
    elif 25.00 > total <= 30.00:
        return "Seu IMC é de : ", total, " Sobrepeso"
    elif 30.00 > total <= 35.00:
        return "Seu IMC é de : ", total, " Obesidade nível 1"
    else:
        return  "Seu IMC é de : ", total, " Obesidade nível 2"

Resultado = CalculaIMC(98, 1.79)

print(Resultado)

File: Php/5_CalculoIMC.php
<?php

function imc($peso, $altura){
    $total = $peso / ($altura * $altura);
    
    if($total <= 1.85) {
        return "Seu IMC é: " . $total . " Você está abaixo do peso - MAGRO";
    } else if ($total > 1.85 && $total <= 25) {
        return "Seu IMC é: " . $total . " Você está com o peso ideal - NORMAL";
    } else if ($total > 25 && $total <= 30) {
        return "Seu IMC é: " . $total . " Você está com o peso acima do ideal - SOBREPESO";
    }  else if ($total > 30 && $total <= 35) {
        return "Seu IMC é: " . $total . " Você está com o peso muito acima do ideal - OBESIDADE NÍVEL 1";
    } else if ($total > 35) {
        return "Seu IMC é: " . $total . " Você está com o peso critico - OBESIDADE NÍVEL 2";
    }
}

echo(imc(98, 1.79));
    
    

?>